### ISSUE TITLE
[Bug] AppBar reported by Windows Narrator in UWP builds

### ISSUE THREAD

**Issue by johnshardman** [xamarin/Xamarin.Forms#12724]
When using Windows Narrator with an app built with XF 4.8 (using NavigationPages), Narrator reports for the AppBar. Narrator should report for ToolbarItems (which it does), but not for the AppBar itself.

This can be seen using the sample at https://github.com/xamarin/xamarin-forms-samples/tree/master/UserInterface/Accessibility - that doesn't have any ToolbarItems, but still reports the AppBar.

With Narrator running, use Narrator's navigation keys to navigate the accessibility tree, not the UI tree.
I use 'o' for next item and 'u' for back item, although the documentation at https://support.microsoft.com/en-us/windows/appendix-b-narrator-keyboard-commands-and-touch-gestures-8bdab3f4-b3e9-4554-7f28-8b15bd37410a lists other keys.

### Basic Information

- Version with issue: 4.8
- IDE: VS2019
- Platform Target Frameworks: 
  - UWP:  18362

**Comment by rmarinho** [comment]
Why wouldn't the app report the AppBar? Is that documented somewhere?

**Comment by johnshardman** [comment]
> Why wouldn't the app report the AppBar?

Users of screen readers want to be told about controls/views that the user wants to interact with, or that provide information to the user. The user interacts with toolbar items. The user might be interested in labels that appear on status bars etc. The AppBar itself is not something that the user will normally interact with or obtain information from directly. Having screen readers report about controls/views that the user does not interact with or obtain information from directly results in "noise" that makes it inefficient/difficult to use screens.

For completeness, some would say that low-vision users (as opposed to completely blind users) might want to be told about anything that they can see as it might be confusing to see something but not be told about it. It's debatable whether the AppBar would fall into that category, as toolbar items etc that appear on the AppBar are reported. For complete flexibility, I would suggest not including the AppBar itself in the accessibility tree, but make private/internal members of the renderer public or protected so that a custom renderer could change that in the rare case where somebody might want it included.

**Comment by rachelkang** [comment]
@johnshardman thanks for submitting this suggestion! Although this isn't a bug (the behavior is same on a normal UWP app), it definitely is a valid feature request that we'll keep an eye on!

**Comment by jfversluis** [comment]
Thanks for this suggestion! As Xamarin.Forms is now in maintenance mode, this will not happen anymore for Xamarin.Forms. We're only adding bugfixes and stability fixes.

If this is still important to you, make sure to check the [.NET MAUI repo](https://github.com/dotnet/maui) and see if it's already on the roadmap. If not, feel free to open a discussion to discuss a change first or open an issue with a detailed feature request. Thanks!
